test/basic.test.js

```diff
@@ -21,11 +21,13 @@ export default class App extends React.Component<Props, State> {
   }
 
   onChange = (selectedItem: Item) => {
-    selectedItem
+    // eslint-disable-next-line no-console
+    console.log('selectedItem', selectedItem)
   }
 
   onUserAction = (changes: StateChangeOptions<Item>) => {
```

Comment:
`onUserAction` isn't used anywhere AFAIK, so it should be safe to remove that?

Reply:
I think this file is used for TypeScript tests, let's leave it :+1: